static/docs/use-cases/index.md

```diff
@@ -1,15 +1,28 @@
 # Use Cases
 
-Here we provide an overview for some DVC use cases, from very basic
-([data and model files management](/doc/use-cases/data-and-model-files-versioning))
-to more advanced (optimizing resources on a
-[single development machine](/doc/use-cases/multiple-data-scientists-on-a-single-machine)).
-This list is _not_ exhaustive. We keep growing it and try to include interesting
-scenarios from the community. Please, [contact us](/support) if you have some
-suggestions!
-
-If you want to get some hands-on experience we recommend to follow along
-[Get Started](/doc/get-started) and try examples at the very end of it. They
-include prepared datasets and code to run. Use cases are not written to be run
-end to end, specifically we don't provide prepared datasets and code to train
-models.
+We provide short articles on common DVC scenarios in this section, with a
```

Comment:
better to say something like: "ML workflow or data management scenarios that DVC can help with or improve"

Reply:
Updating